doc/flatpak-config.xml

```diff
@@ -70,7 +70,8 @@
                 <listitem><para>
                    This key is used when languages is not set, and it defines extra locale
                    extensions on top of the system configured languages. The value is a
-                   semicolon-separated list of two-letter language codes.
+                   semicolon-separated list of two-letter language codes and 5-to-10-letter
+                   locale codes.
```

Comment:
I think the 5-to-10 letter qualifier here is less than helpful. I would suggest: 'list of two-letter language codes or locale identifiers (e.g. en_DK.utf8)'